src/styles/components/login-modal.less

```diff
@@ -0,0 +1,133 @@
+.modal {
+
+  &.login-modal {
+
+    .modal-content {
+      height: auto;
+      overflow: visible;
+      position: relative;
+
+      .container {
+        width: 100%;
+      }
+    }
+
+    .modal-header {
+      background: transparent;
+      height: 144px;
+      padding: 0;
+
+      .container {
+        padding: 0;
+      }
+    }
+
+    .dcos-logo {
+      left: 50%;
+      position: absolute;
+      top: -55px;
+      transform: translateX(-50%);
+    }
+
+    .modal-body {
+      padding-bottom: 0;
+      padding-right: 0;
+      padding-top: @base-spacing-unit * 2/3;
+    }
+
+    .login-modal-heading {
+      color: @neutral;
+    }
+
+    .login-modal-content {
+      padding-bottom: @modal-body-padding-vertical;
+      padding-right: @modal-body-padding-horizontal;
+    }
+  }
+}
+
+& when (@screen-mini-enabled) {
+
+  @media (min-width: @screen-mini) {
+
+    .modal {
+
+      &.login-modal {
+
+        .modal-body {
+          padding-top: @base-spacing-unit-screen-mini * 2/3;
+        }
+      }
+    }
+  }
+
+}
+
+& when (@screen-small-enabled) {
+
+  @media (min-width: @screen-small) {
+
+    .modal {
+
+      &.login-modal {
+
+        .modal-body {
+          padding-top: @base-spacing-unit-screen-small * 2/3;
+        }
+
+        .login-modal-content {
+          padding-bottom: @modal-body-padding-vertical-screen-small;
```

Comment:
why do we use `@modal-body-padding-vertical-screen-small` for `padding-bottom` but `@base-spacing-unit-screen-small` for `-top`?